### PR TITLE
Add Makefile to init template

### PR DESCRIPTION
**Pull request by fracek**
As mentioned in #6, I added a Makefile to the init template.

I also changed the filename of the test file so that pytest collects it automatically.

### CODE REVIEW

**Comment by martriay** [comment]
Thanks! Looks good, would you mind adding some Makefile documentation on the README?

**Review by martriay — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,10 +33,8 @@ nile init
 ✅ Dependencies successfully installed
 🗄  Creating project directory tree
 ⛵️ Nile project ready! Try running:
-
-nile compile
```
Comment (fracek) [inline README.md:37]: I think this was a runaway line. Let me know if it was there for a reason.

**Comment by fracek** [comment]
I added a line to the readme.

File: README.md (hunk 1 above)
Comment (martriay) [reply README.md:37]: it was meant to work as an example command you can run with nile (notice the colon in the line above)

**Pull request merged**